tests/props.spec.ts

```diff
@@ -342,7 +343,7 @@ describe('props', () => {
     })
 
     it('should get props from functional component', async () => {
-      const wrapper = mount(Title, {
+      const wrapper: VueWrapper<any> = mount(Title, {
```

Comment:
I had to add this to make the compilation pass; I'm not sure I understand why this isn't working properly with the functional component?

Reply:
I'm not sure.

@johnsoncodehk Would you mind taking a look at this PR please? Is this the proper way to type props?

Reply:
If we add something that requires users to add code to their code base (like this) I think this would be a breaking change, which I'm not sure we'd want to do.

Not sure if Johnson will have time to 👀 , we may need to have a look. Vue types are too complex 😓

Reply:
Okay have applied @pikax 's suggestion and the build is green without this cast